server/controllers/reports_proposed/data/balance_sheet.js

```diff
@@ -1,5 +1,11 @@
-// reports_proposed/data/balance_sheet.js
```

Comment:
This report should include every account, even income expense, it is called a provisional balance, because it is generated before closing the fiscal year.